src/Servers/Kestrel/Core/src/CoreStrings.resx

```diff
@@ -665,6 +665,9 @@ For more information on configuring HTTPS see https://go.microsoft.com/fwlink/?l
   <data name="Http3ControlStreamErrorInitializingOutbound" xml:space="preserve">
     <value>Error initializing outbound control stream.</value>
   </data>
+  <data name="Http3WebTransportStatusMismatch" xml:space="preserve">
+    <value>HTTP/3 webtransport negotiation mismatch. Currently client has it '{clientStatus}' and server has it '{serverStatus}'</value>
```

Comment:
Something more like: `Client support: '{clientStatus}'. Server support: '{serverStatus}'`

Reply:
I just wanted to be consistent with the following [message](https://github.com/dotnet/aspnetcore/blob/90ee8e4a019faa4a9d2b405be4aa76957599377e/src/Servers/Kestrel/Core/src/CoreStrings.resx#L671) which was included with webtransport implementation

```
<data name="Http3DatagramStatusMismatch" xml:space="preserve">
    <value>HTTP/3 datagrams negotiation mismatch. Currently client has it '{clientStatus}' and server has it '{serverStatus}'</value>
  </data>
``` 
Whether you find it better or not is entirely up to you, and I'm open to making adjustments accordingly.

Reply:
We should change that one too 😆 
And Http3WebTransportStatusMismatch

Reply:
done